esp_lcd_qemu_rgb/examples/lcd_qemu_rgb_panel/main/lcd_qemu_rgb_panel_main.c

```diff
@@ -120,7 +117,8 @@ void app_main(void)
     esp_lcd_panel_handle_t panel_handle = NULL;
     esp_lcd_rgb_qemu_config_t panel_config = {
         .width = EXAMPLE_LCD_H_RES,
-        .height = EXAMPLE_LCD_V_RES
+        .height = EXAMPLE_LCD_V_RES,
+        .bpp = RGB_QEMU_BPP_32,
```

Comment:
Given that the CONFIG_LV_COLOR_DEPTH_32  check above was removed, you probably need to make the bpp value here dependent on `CONFIG_LV_COLOR_DEPTH_*`

Reply:
I have added this check back.